nemo/export/tensorrt_llm.py

```diff
@@ -455,13 +461,15 @@ def export(
                         fp8_kvcache=fp8_kvcache,
                     )
 
-                    for weight_dict, model_config in zip(weights_dicts, model_configs):
+                    for trtllm_model_weights, trtllm_model_config in zip(
+                        trtllm_model_weights_list, trtllm_model_config_list
+                    ):
                         build_and_save_engine(
                             max_input_len=max_input_len,
                             max_output_len=max_output_len,
                             max_batch_size=max_batch_size,
-                            model_config=model_config,
```

Comment:
These changes are renames so that we don't override `model_config` that I need below in `_export_to_nim_format`